src/pages/index.js

```diff
@@ -30,72 +32,219 @@ const previewText = previewImageModal.querySelector(".modal__preview-text");
 
 const cardList = document.querySelector(".cards__list");
 
+const avatarModal = document.querySelector("#avatar-modal");
+const profileAvatarButton = document.querySelector(
+  ".profile__pic-overlay_button"
+);
+
+// API
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "6e53887a-44e7-4719-8efe-74af2f636950",
+    "Content-Type": "application/json",
+  },
+});
+
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([data, initialCards]) => {
+    userInfo.setUserInfo(data);
+    userInfo.setAvatar(data.avatar);
+    section = new Section(
+      {
+        items: initialCards,
+        renderer: (item) => {
+          const cardEl = renderCard(item);
+        },
+      },
+      ".cards__list"
+    );
+    section.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+let section;
+
 // Validation //
 
 const editFormElement = document.querySelector("#edit-form");
 const addFormElement = document.querySelector("#add-form");
+const avatarFormelement = document.querySelector("#avatar-form");
 
 const editFormValidator = new FormValidator(config, editFormElement);
 const addFormValidator = new FormValidator(config, addFormElement);
+const avatarFormValidator = new FormValidator(config, avatarFormelement);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```